CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 7.0.0 (2022-04-23)
+
+This version contains **breaking** changes, as bugsnag-unity has been updated to use a more advanced Exception interceptor and existing grouping between errors in the Bugsnag dashboard may be broken.
```

Comment:
"broken" suggests not working, whereas we mean it has changed. How about:

```suggestion
This version contains **breaking** changes in the grouping of errors on the Bugsnag dashboard due to the introduction of a new exception interceptor that directly reports the thrown C# exceptions, rather than obtaining this information from Unity's logging mechanism. 
```

Reply:
```suggestion
This version contains **breaking** changes: some errors generated by the new version will no longer group in the Bugnsag dashboard with equivalent errors generated by older versions. This is due to the introduction of a new exception interceptor that intercepts the C# exceptions objects, rather than parsing the information from Unity logs.
```